### PR TITLE
Allowing Optional Columns for AddAirtableRow

### PR DESCRIPTION
**Pull request by btmoore4**

### CODE REVIEW

File: src/routes/api/getColumnMapping.js [hunk 1]
```diff
@@ -1,11 +0,0 @@
-'use strict';
```
Comment (brainomite) [inline src/routes/api/getColumnMapping.js:1]: Strongly suggest keeping this. We'll need it for the next phase when we do syncing of multiple tables and or bases, When we handle multiple, mappings, system variables will no longer cut it and we will need a data entry method for a database of some sort

File: src/routes/api/getColumnMapping.js [hunk 2]
```diff
@@ -1,11 +0,0 @@
...
-
-module.exports = (app) => {
```
Comment (brainomite) [inline src/routes/api/getColumnMapping.js:3]: Strongly suggest keeping this. We'll need it for the next phase when we do syncing of multiple tables and or bases, When we handle multiple, mappings, system variables will no longer cut it and we will need a data entry method for a database of some sort

File: src/routes/api/setColumnMapping.js [hunk 3]
```diff
@@ -1,27 +0,0 @@
-'use strict';
```
Comment (brainomite) [inline src/routes/api/setColumnMapping.js:1]: Strongly suggest keeping this. We'll need it for the next phase when we do syncing of multiple tables and or bases, When we handle multiple, mappings, system variables will no longer cut it and we will need a data entry method for a database of some sort

File: src/service/airtablePoller.js [hunk 4]
```diff
@@ -4,7 +4,6 @@ const Airtable = require('airtable');
 const ChangeDetector = require('airtable-change-detector');
```
Comment (brainomite) [inline src/service/airtablePoller.js:4]: New logic introduced, but no mocha unit tests were created to test the logic

File: src/routes/api/getColumnMapping.js (hunk 1 above)
Comment (btmoore4) [reply src/routes/api/getColumnMapping.js]: re-added

File: src/routes/api/setColumnMapping.js (hunk 3 above)
Comment (btmoore4) [reply src/routes/api/setColumnMapping.js]: re-added

File: test/routes/api/setColumnMapping.test.js [hunk 5]
```diff
@@ -1,18 +0,0 @@
-'use strict';
-const request = require('supertest');
```
[the remark this answers is not in the document]
Comment (btmoore4) [reply test/routes/api/setColumnMapping.test.js]: re-added

File: test/routes/api/getColumnMapping.test.js [hunk 6]
```diff
@@ -1,18 +0,0 @@
-'use strict';
```
[the remark this answers is not in the document]
Comment (btmoore4) [reply test/routes/api/getColumnMapping.test.js]: re-added

**Pull request merged**